### ISSUE TITLE
Router isn't good

### ISSUE THREAD

**Issue by delgec** [ryansolid/solid-realworld#24]
why didn't you use Solid-Router?

Can a fork be done for this?

Cheers.

**Comment by ryansolid** [comment]
Yeah this needs an update.  I built this over 3 years ago when I was still figuring out routing patterns. This example is really out of date.

My intention was to do so when SolidStart was in a better place because as we are being a bit more prescriptive on best patterns for data fetching and loading with the next version of the Router.

**Comment by delgec** [comment]
> Yeah this needs an update. I built this over 3 years ago when I was still figuring out routing patterns. This example is really out of date.
> 
> My intention was to do so when SolidStart was in a better place because as we are being a bit more prescriptive on best patterns for data fetching and loading with the next version of the Router.



> Yeah this needs an update. I built this over 3 years ago when I was still figuring out routing patterns. This example is really out of date.
> 
> My intention was to do so when SolidStart was in a better place because as we are being a bit more prescriptive on best patterns for data fetching and loading with the next version of the Router.


I am a junior developer and I need starter kits from the seniors for the best practies.

Is it possible to update this source code?

Thanks a lot
@ryansolid
